### ISSUE TITLE
Other Plans- Screen reader reads only contract number and date

### ISSUE THREAD

**Issue by Jacob15** [veshu/accessibility-bughunt-bluesource#75]
**Describe the bug**
Screen reader reads only contract number and date in other plans popup

**To Reproduce**
1. Go to 'http://localhost:4200/find-care'
2. Click on 'Other Plans'
3. Scroll down to 'Plans details'
4. Screen reader reads only contract number and date. It doesn't read out other highlighted text.

**Expected behavior**
Screen reader to read all the plan details Coverage, Status, Relationship to Policyholder etc.

**Device**: Laptop
**Operating System**: Windows 10
**Browser(with version)**: Edge

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Tools used**: Screen Reader in Windows 10

**Attachments**
https://drive.google.com/open?id=1XjgXSzoMrszzssXNlEAkbmNdzubTmBzJ

**Comment by Jacob15** [comment]
Link to Video:-
https://drive.google.com/open?id=1KBkhcsufRGpn6TlT_3voEk1Z7z0e5QP-

**Comment by drasticdpk** [comment]
WCAG Category: 4. Robust
WCAG Level: A
WCAG Success Criterion: 4.1.2 Name, Role, Value
